docs/data-guidelines.md

```diff
@@ -0,0 +1,159 @@
+# Data guidelines
+
+This file contains recommendations to help you record data in a consistent and understandable way. It covers the project's preferences for the way features should be represented, rather than hard requirements encoded in the schema definitions or linter logic.
+
+- [Data guidelines](#data-guidelines)
+  * [Constructors](#constructors)
+  * [DOM events (`eventname_event`)](#dom-events---eventname-event--)
+  * [Secure context required (`secure_context_required`)](#secure-context-required---secure-context-required--)
+  * [Web Workers (`worker_support`)](#web-workers---worker-support--)
+  * [Non-functional defined names imply `partial_implementation`](#non-functional-defined-names-imply--partial-implementation-)
+  * [Release lines and backported features](#release-lines-and-backported-features)
+  * [Node.js releases before 1.0](#nodejs-releases-before-10)
+  * [Safari for iOS versioning](#safari-for-ios-versioning)
+
+<!-- BEGIN TEMPLATE
+
+## Short title in sentence case
+
+A description of what to do, preferably in the imperative. If applicable, include an example to illustrate the rule.
+
+If it's helpful to understanding the rule, summarize the rationale. Definitely cite the issue or pull request where this was decided (it may be the PR that merges the policy).
+
+-- END TEMPLATE -->
+
+## Constructors
+
+Name a constructor for an API feature the same as the parent feature (unless the constructor doesn't share the name of its parent feature) and have a description with text in the form of `<code>Name()</code> constructor`.
+
+For example, the `ImageData` constructor, `ImageData()`, is represented as `api.ImageData.ImageData`. It has the description `<code>ImageData()</code> constructor`, like this:
+
+```json
+{
+  "api": {
+    "ImageData": {
+      "__compat": {},
+      "ImageData": {
+        "__compat": {
+          "description": "<code>ImageData()</code> constructor",
+          "support": {}
+        }
+      }
+    }
+  }
+}
+```
+
+
+## DOM events (`eventname_event`)
+
+Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
+
+For example, the feature for a `focus` event would be named `focus_event` with the description text `<code>focus</code> event`, like this:
+
+```json
+{
+  "api": {
+    "Element": {
+      "__compat": {},
+      ...
```

Comment:
"..." renders weirdly. Can we do something about that or just remove it?

Reply:
👍 I removed it